packages/kuma-gui/src/test-support/index.ts

```diff
@@ -97,7 +97,11 @@ export const handler = (fs: FS, env: AEnv) => {
         method: request.method,
         url: new URL(request.url),
         body: request.body ? JSON.parse(await new Response(request.body).text() || '{}') : {},
-        params,
+        params: Object.fromEntries(
+          Object.entries(params).filter(
+            (entry): entry is [string, string | readonly string[]] => typeof entry[1] !== 'undefined',
+          ),
+        ),
```

Comment:
The type of `PathParam` has changed to

```typescript
type PathParams<KeyType extends keyof any = string> = {
    [ParamName in KeyType]?: string | ReadonlyArray<string>;
};
```

which caused that there was a type mismatch

```shell
Type 'PathParams' is not assignable to type 'Record<string, string | readonly string[]>'.
  'string' index signatures are incompatible.
    Type 'string | readonly string[] | undefined' is not assignable to type 'string | readonly string[]'.
      Type 'undefined' is not assignable to type 'string | readonly string[]'.
```

To make sure that we can continue to safely access the values (without conditional chaining, e.g. `name?.includes()`) I added this to filter for params with values of `undefined`.